crypto/ecdsa/utils.go

```diff
@@ -0,0 +1,73 @@
+package ecdsa
+
+import (
+	"bufio"
+	"crypto/ecdsa"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/google/uuid"
+)
+
+func WriteKey(path string, privateKey *ecdsa.PrivateKey, password string) error {
+	UUID, err := uuid.NewRandom()
+	if err != nil {
+		return err
+	}
+	key := &keystore.Key{
+		Id:         UUID,
```

Comment:
What is this UUID for? And why isn't it part of our privateKey struct? Seems weird that if we read/write/read/write a privateKey we end up writing with different UUIDs. Is it just useless? If so, add a comment

Reply:
- UUID is required by the geth keystore encrypt function. it's just a unique identifier they use to store the keys 
 - we don't have a `privateKey` struct for ecdsa key. we have for BLS. here we are just using the crypto/ecdsa key type.
 - We are just providing a utility to store a private key this path. right the the caller of this is handling the override or not. we could introduce that validation in our library

Reply:
Sounds good. Please do add a comment with this information.